### ISSUE TITLE
Fix Accessibility w/ Color Contrast on Disabled `SearchableDropdown`

### ISSUE THREAD

**Issue by jcq** [department-of-veterans-affairs/caseflow#14878]
## Description
We are seeing a warning regarding insufficient color contrast between foreground text and background color when `SearchableDropdown` is set to disabled/readonly. Let's get guidance from Design as to what the styling should be, and implement accordingly.

## Acceptance criteria
- [ ] `SearchableDropdown` passes axe tests when `readOnly` prop is `true` 

## Background/context/resources
One can see both the behavior and the results of the accessibility tests [in the "Read Only" story](https://www.caseflowdemo.com/storybook/?path=/story/commons-components-searchabledropdown--read-only) for `SearchableDropdown` in Storybook

PR #14229 from May 2020 seems related, and likely set styles that were lost during the refactoring of `SearchableDropdown` to use the latest version of `react-select`.

## Technical notes
Adjustments to this can be made in `_select-dropdown.scss`

**Comment by rutvigupta-design** [comment]
From Rutvi and Monique: we'd like the foreground text color to be` #323a45` – please let me know if there's anything else we can specify!

**Comment by jcq** [comment]
### Old version

![Screen Shot 2020-09-18 at 12 08 52](https://user-images.githubusercontent.com/2581274/93636077-22aff400-f9a8-11ea-8d90-b54c0ab29832.png)

### Updated version
![Screen Shot 2020-09-18 at 12 08 28](https://user-images.githubusercontent.com/2581274/93636115-35c2c400-f9a8-11ea-97c3-a97706431730.png)
